cf_xarray/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .accessor import CFAccessor  # noqa
+from .geometry import cf_to_geometry, geometry_to_cf  # noqa
```

Comment:
Should we call these `cf_to_shapely` and `shapely_to_cf` instead?

Or `cf_geometry_to_shapely` and `shapely_to_cf_geometry`

Reply:
Let's go with `cf_to_shapely`.